ERCS/erc-7824.md

```diff
@@ -0,0 +1,332 @@
+---
+eip: 7824
+title: Nostr and NFT Compatible Social Media Standard
```

Comment:
```suggestion
title: Social Media NFTs
```
Standard is superfluous in an ERC title.
Is this ERC specific to Nostr protocol?  If not, then suggest using a wider title.